server/src/logger.js

```diff
@@ -0,0 +1,34 @@
+require("dotenv").config();
+const winston = require("winston");
+const SlackHook = require("winston-slack-webhook-transport");
+
+const transports = {
+  console: new winston.transports.Console({
+    format: winston.format.simple(),
+  }),
+  slack: new SlackHook({
+    level: "warn",
+    format: winston.format.simple(),
+    webhookUrl: process.env.SLACK_WEBHOOK_URL,
+  }),
+};
+
+const logger = winston.createLogger({
+  transports: [transports.console],
+});
+
+logger.stream = {
```

Comment:
What would be the practice of invoking this from the main application? Would it just be listening to the logging stream for any `warn` level messages?

Reply:
Yes exactly.